src/components/maincontroller.ts

```diff
@@ -463,12 +469,12 @@ export function changeStream(ticks, params) {
 // TODO save namespace somewhere global?
 window.castReceiverContext.addCustomMessageListener(
     'urn:x-cast:com.connectsdk',
-    function (evt) {
-        var data = evt.data;
+    function (evt: any) {
+        let data: any = evt.data;
 
         // Apparently chromium likes to pass it as json, not as object.
         // chrome on android works fine
-        if (typeof data == 'string' || data instanceof String) {
+        if (typeof data == 'string') {
```

Comment:
```suggestion
        if (typeof data === 'string') {
```

If it doesn't exist, we should make a rule in this repo for enforcing this.